lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -503,6 +505,39 @@ def _configure_node(
 
                 node_context.data_disks.append(data_disk)
 
+        # Set device passthrough params for node context
+        if node_runbook.device_passthrough:
+            self._check_if_host_support_iommu(self.host_node)
+            dev_conf_list = node_runbook.device_passthrough
```

Comment:
is it `dev_conf_list = node_runbook.device_passthrough_configs`

Reply:
There is no `device_passthrough_configs `in the runbook schema.

```
# Configuration options for device-passthrough.
device_passthrough: Optional[List[LibvirtHostPciDeviceSchema]] = None
```

This is the parent to attribute in yaml to pass list of devices

Reply:
`device_passthrough_configs` is something we set for node_context